miner/src/block_assembler.rs

```diff
@@ -489,18 +484,17 @@ mod tests {
             // uncles_count_limit,
         } = block_template;
 
-        let (cellbase_id, cellbase) = {
-            let CellbaseTemplate { hash, data, .. } = cellbase;
-            (hash, data)
+        let cellbase = {
+            let CellbaseTemplate { data, .. } = cellbase;
```

Comment:
Do we still need `CellbaseTemplate` after removing cellbase id from header? Can we add it to `Vec<TransactionTemplate>` ?

Reply:
```
pub struct TransactionTemplate {
    pub hash: H256,
    pub required: bool,
    pub cycles: Option<Cycle>,
    pub depends: Option<Vec<u32>>,
    pub data: Transaction, // temporary
}
```
`cellbase` has no depends, and it's required no doubt, so perfer keep it.